src/Command/SSL/LetsencryptJobStatusCommand.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Pagely\AtomicClient\Command\SSL;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class LetsencryptJobStatusCommand extends AbstractSSLCmd
+{
+    protected $commandName = 'ssl:le:jobstatus';
+
+    public function configure()
+    {
+        parent::configure();
+        $this->setDescription('Check letsencrypt job status')
+            ->addArgument('id', InputArgument::REQUIRED, 'ID of the letsencrypt job (not jobId)')
```

Comment:
is the text `(not jobId)` supposed to say `(not appid)` ?

Reply:
It is valid... there is a Pagely job queue id (jobId) and a letsencrypt job id.